SPECS/hwloc/hwloc.spec

```diff
@@ -70,6 +70,12 @@ Requires:       %{name}-plugins%{?_isa} = %{version}-%{release}
 
 %prep
 %autosetup -p1
+%if %{with_check}
+#Remove the failing test 
+sed -i '/hwloc_get_area_memlocation/d' tests/hwloc/Makefile.am
+#updating Makefile.am needs a rerun of autoreconf
+autoreconf 
+%endif
```

Comment:
I'd even run this always, so that we are sure to have the same build with and without the tests.
```suggestion
# Remove the failing test 
sed -i '/hwloc_get_area_memlocation/d' tests/hwloc/Makefile.am

# Updating Makefile.am needs a rerun of autoreconf
autoreconf 
```